apps/web/playwright/booking-seats.e2e.ts

```diff
@@ -231,6 +252,7 @@ testBothBookers.describe("Reschedule for booking with seats", () => {
 
     await expect(page).toHaveURL(/.*booking/);
 
+    await page.waitForTimeout(300);
```

Comment:
Looks like between page navigation and playwright execution I should sleep a little before getting data on the DB

Reply:
This is flaky, instead consider waitForURL